slices/dpkg.yaml

```
@@ -14,19 +18,42 @@ slices:
       - tar_tar
       - zlib1g_libs
     contents:
-      /usr/bin/dpkg*:
+      /usr/bin/dpkg:
+      /usr/bin/dpkg-deb:
+      /usr/bin/dpkg-divert:
+      /usr/bin/dpkg-maintscript-helper:
+      /usr/bin/dpkg-query:
+      /usr/bin/dpkg-realpath:
+      /usr/bin/dpkg-split:
+      /usr/bin/dpkg-statoverride:
+      /usr/bin/dpkg-trigger:
       /usr/bin/update-alternatives:
       /usr/libexec/dpkg/*:
       /usr/sbin/start-stop-daemon:
-      /var/lib/dpkg/tmp.ci/: {make: true}
-      /var/log/dpkg.log: {text: ""}
+
+  copyright:
+    contents:
+      /usr/share/doc/dpkg/copyright:
 
   config:
     contents:
       /etc/dpkg/dpkg.cfg:
+      /etc/dpkg/dpkg.cfg.d/: {make: true}
 
   locales:
     essential:
       - dpkg_bins
     contents:
       /usr/share/locale/**/dpkg.mo:
+
+  tables:
+    contents:
+      /usr/share/dpkg/*table:
+
+  var:
+    # Directories that are included in the tarball because they need to exist in /var
+    contents:
+      /var/lib/dpkg/alternatives/: {make: true}
+      /var/lib/dpkg/info/: {make: true}
+      /var/lib/dpkg/parts/: {make: true}
+      /var/lib/dpkg/updates/: {make: true}
```

```suggestion
    # Directories that are included in the tarball because they need to exist in /var
    contents:
      /var/lib/dpkg/alternatives/:
      /var/lib/dpkg/info/:
      /var/lib/dpkg/parts/:
      /var/lib/dpkg/updates/:
```

They actually exist in the deb, so you don't need to explicitly `make` them

```
drwxr-xr-x root/root         0 2024-07-17 09:09 ./var/lib/dpkg/alternatives/
drwxr-xr-x root/root         0 2024-07-17 09:09 ./var/lib/dpkg/info/
drwxr-xr-x root/root         0 2024-07-17 09:09 ./var/lib/dpkg/parts/
drwxr-xr-x root/root         0 2024-07-17 09:09 ./var/lib/dpkg/updates/
```

---

Oh, neat - didn't realise that. Thanks!